### DISCUSSION TITLE
Lost account, with no authenticator

### DISCUSSION THREAD

**Discussion by akinekine** [community/community#78537] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, dear support and forum, I was wondering if its possible to restore an account where I have forgotten password and lost auth on.

I have recently made another account with basically a clone of my original, but I still have my domain pointing at it and therefore can't control the cert from Cloudflare. Can you please let me know if its possible to recover that account? 

My icon is the same on both accounts, and I have the device model the same as when I used that account, but in short, I have lost the device, reset from cloud, and wasn't allowed to make a full download of the backup at one point in the reset process. So now I dont have access to 2FA, I panicked and removed the information from the other account and then lost access. Is anyone at github able to step in to help me recoved my domain. It's really important to me.

Thank you

**Comment by kale1d0code** [comment]
I am having the same issue,
I am unable to setup a github account for my work email address because the email address is already in use, however there is no way to login to this account. advise on how to delete or access the account would be invaluable
